pkg/sink/sink.go

```diff
@@ -53,25 +67,42 @@ func (r Resource) HandleEvent(response http.ResponseWriter, request *http.Reques
 		return
 	}
 
+	log.Printf("EventListener: %s in Namespace: %s handling event with payload: %s and header: %v",
+		r.EventListenerName, r.EventListenerNamespace, string(event), request.Header)
```

Comment:
nit: there is a log.Printf and a fmt.Printf a couple of lines below with the same info. We should remove one.

Reply:
log is being used to log here and fmt.Fprintf is being used to send a response to the event provider.